### PR TITLE
Add a link to the FireCloud Status page in the footer [GAWB-3351]

### PR DESCRIPTION
**Pull request by ghost**
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [x] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by aweng98** [comment]
Is this async-loading UI object that may causes page to move when finished rendering on page? If so, I'm concerned it causes UI automation tests to become flaky.

File: src/cljs/main/broadfcui/main.cljs [hunk 1]
```diff
@@ -239,7 +239,7 @@
                                                      "FireCloud service is temporarily unavailable.")]
              (notifications/render-alert {:title title
                                           :message msg
-                                          :link "http://status.firecloud.org/"
+                                          :link (config/status-url)
```
Comment (ghost) [inline src/cljs/main/broadfcui/main.cljs:242]: @aweng98 are you referring to this change specifically? Please let me know, I'm happy to revert if the alternative might cause flakiness.

**Comment by ghost** [comment]
jenkins retest

**Comment by jmthibault79** [comment]
@jbuabbud Should probably close this until the new version based on https://github.com/broadinstitute/firecloud-develop/pull/1102 is ready - so we're not reviewing outdated code

**Comment by ghost** [comment]
Closing until broadinstitute/firecloud-develop#1102 merges.

**Review by jmthibault79 — APPROVED** [review]
Tested locally: no status link and no rendering errors.

Locally with a manual update to config.json adding a statusUrl: Status link is present with my value.

**Pull request merged**